### ISSUE TITLE
No such file or directory - versioneer error.

### ISSUE THREAD

**Issue by useraccessdenied** [cryfs/homebrew-tap#12]
Mac OS version: 12.3.1
cryfs version: 11.2

❯ brew install cryfs/tap/cryfs
==> Downloading https://github.com/cryfs/cryfs/releases/download/0.11.2/cryfs-0.11.2.tar.xz
Already downloaded: /Users/kapil/Library/Caches/Homebrew/downloads/9dc7238113b42752def2773c94a3e21cd132b0bccba20d0fd6bfe34f9a84f648--cryfs-0.11.2.tar.xz
==> Installing cryfs from cryfs/tap
==> cmake . -GNinja -DBUILD_TESTING=off
Last 15 lines from /Users/kapil/Library/Logs/Homebrew/cryfs/01.cmake:
-- Compiler: /opt/homebrew/Library/Homebrew/shims/mac/super/clang++
-- Compiler options:
-- Compiler definitions:
-- Build type: Release
-- Building version
CMake Error at src/gitversion/gitversion.cmake:13 (MESSAGE):
  Error running versioneer.  Return code is: No such file or directory, error
  message is:
Call Stack (most recent call first):
  src/gitversion/CMakeLists.txt:4 (get_git_version)

-- Configuring incomplete, errors occurred!
See also "/tmp/cryfs-20220409-19009-hp1r2m/CMakeFiles/CMakeOutput.log".
See also "/tmp/cryfs-20220409-19009-hp1r2m/CMakeFiles/CMakeError.log".

Cmake log file is attached. If anyone can make sense of it, it would be helpful. Error's at line 295
[01.cmake.log](https://github.com/cryfs/homebrew-tap/files/8457446/01.cmake.log)

**Comment by marshevms** [comment]
Hi! For now you can try this `brew install cryfs/tap/cryfs --HEAD`. At your own risk!

**Comment by klarrio-dlamb** [comment]
Nice one - thanks @marshevms!
 
I happened to be digging into this this evening.  I originally had M1 OpenMP problems similar to https://github.com/cryfs/homebrew-tap/issues/10 and parked it a couple of times.  Tried again tonight, and these seem to be resolved by more recent dependency updates, then hit this.

I'd mostly figured out why, then somehow stumbled on your recent issue / release request.

For @useraccessdenied or anyone else's benefit, the error is due to Apple removing python 2 -> the `python` binary disappeared in [Monterey 12.3](https://developer.apple.com/documentation/macos-release-notes/macos-12_3-release-notes).  This is the `No such file or directory` error.

https://github.com/cryfs/cryfs/pull/413 wasn't created because of Apple, but fixes this by explicitly referring to python3.  Although it was merged in January, it didn't get included in the 0.11.2 release.

I had first tried linking my homebrew python3 to `python` which worked for manual builds, but didn't work for homebrew builds.
